<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productsForm</class>
 <widget class="QWidget" name="productsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="content">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="productsTab">
      <attribute name="title">
       <string>Products</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="productToolsWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="search">
            <property name="placeholderText">
             <string>Search</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="addBtn">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/add.png</normaloff>:/Icons/res/add.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteBtn">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/delete.png</normaloff>:/Icons/res/delete.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="productsTabel"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="familysTab">
      <attribute name="title">
       <string>Familys</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="familySearch">
            <property name="placeholderText">
             <string>Search</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/add.png</normaloff>:/Icons/res/add.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/delete.png</normaloff>:/Icons/res/delete.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableView" name="familysTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="categorysTab">
      <attribute name="title">
       <string>Categorys</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QTableView" name="categorysTable"/>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="placeholderText">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/add.png</normaloff>:/Icons/res/add.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Icons/res/delete.png</normaloff>:/Icons/res/delete.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
